migrations/Version20220106102125.php

```diff
@@ -0,0 +1,31 @@
+<?php
```

Comment:
You can delete the migration file it is not useful otherwise it creates conflicts with other migration files